Objective-C/Internal/CBLDatabase+Internal.h

```diff
@@ -37,6 +37,8 @@
 #import "CBLURLEndpointListener.h"
 #endif
 
+#define msec 1000.0
```

Comment:
Is this required?

Reply:
I guess, no. we can move to CBLCollection.m, since we have already redirected the calls.